Dockerfile

```diff
@@ -0,0 +1,104 @@
+FROM python:3.12-slim-bookworm AS base-libs
+
+LABEL maintainer="mapproxy.org"
+
+RUN apt update && apt -y install --no-install-recommends \
+  python3-pil \
+  python3-yaml \
+  python3-pyproj \
+  libgeos-dev \
+  python3-lxml \
+  libgdal-dev \
+  python3-shapely \
+  libxml2-dev \
+  libxslt-dev && \
+  apt-get -y --purge autoremove && \
+  apt-get clean && \
+  rm -rf /var/lib/apt/lists/*
+
+
+FROM base-libs AS builder
+
+RUN mkdir /mapproxy
+
+WORKDIR /mapproxy
+
+COPY setup.py README.md CHANGES.txt MANIFEST.in ./
+COPY mapproxy mapproxy
```

Comment:
These are the only files that will be included in the wheel. Maybe we need some more?

Reply:
License at least